linera-views/src/memory.rs

```diff
@@ -126,6 +133,10 @@ pub enum MemoryContextError {
     /// Serialization error with BCS.
     #[error("BCS error: {0}")]
     Bcs(#[from] bcs::Error),
+
+    /// The database is not coherent
```

Comment:
```suggestion
    /// The database is not consistent
```

(Not sure about the best term either, but it should match the identifier.)